crates/ruff_db/src/files.rs

```diff
@@ -162,6 +203,12 @@ impl Files {
             let file = entry.value();
             file.sync(db);
         }
+
+        let routes = inner.roots.read().unwrap();
```

Comment:
```suggestion
        let roots = inner.roots.read().unwrap();
```

Reply:
That's what you get for using Copilot lol